tests/network_test.cpp

```diff
@@ -125,15 +125,22 @@ TEST_F(NetworkTest, send_pbft_block) {
   auto nodes = launch_nodes(node_cfgs, 1);
   auto nw1 = nodes[0]->getNetwork();
   auto nw2 = nodes[1]->getNetwork();
+  EXPECT_EQ(1, nw1->getPeerCount());
+  EXPECT_EQ(1, nw2->getPeerCount());
```

Comment:
launch_nodes internally already checks if nodes are connected so no need to check here as well

Reply:
removed